package.json

```diff
@@ -94,7 +94,9 @@
     "yarn": "^1.22.19"
   },
   "dependencies": {
-    "@patternfly/react-core": "^4.267.6",
+    "@patternfly/quickstarts": "^2.3.3",
+    "@patternfly/react-catalog-view-extension": "4.93.15",
+    "@patternfly/react-core": "4.267.6",
```

Comment:
Hey @andrewazores, do u think there is any issue with fixing the version of PF core? I am seeing OCP console is also doing this.

Reply:
That seems like it should be fine.